.github/workflows/lint-and-unit.yml

```diff
@@ -0,0 +1,55 @@
+name: Tox Lint and Unit Tests
+
+on:
+  push:
+    branches-ignore:
+      - master
+
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8"]
```

Comment:
Any reason why we don't also test 3.9-3.11?

Reply:
We don't support this with the python armada client due to issues with airflow, running them on versions any higher than this will error.